src/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
+  has_many :rentals
```

Comment:
- [ ] the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`. They should be implemented according to this [lesson](`tps://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`). Kindly take a careful look at the requirements.